### PR TITLE
Pygame bomberman

### PR DESCRIPTION
**Pull request by Viri0x**
Add reference for a bomberman game with pygame.

### CODE REVIEW

File: mu-tutoriel.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
+# Comment utiliser Mu Editor ?
```
Comment (alarsyo) [inline mu-tutoriel.md:1]: Your PR seems to have a lot of unrelated files in the diff, you should probably rebase on master / delete unrelated commits

**Review by alarsyo — CHANGES_REQUESTED** [review]

**Comment by kolowy** [comment]
Migrated to https://gitlab.com/prologin/gcc/contenus/-/merge_requests/37
